cadc-registry-server/reg-resource-caps.properties

```diff
@@ -142,6 +143,7 @@ ivo://cadc.nrc.ca/maq/XMM = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/maq
 ivo://cadc.nrc.ca/test/ALMA = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/maq/sandbox/capabilities
 ivo://cadc.nrc.ca/test/ESO = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/maq/sandbox/capabilities
 ivo://cadc.nrc.ca/test/GEMINI = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/maq/sandbox/capabilities
+ivo://cadc.nrc.ca/test/GEMINIPROC = https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/reg/maq/sandbox/capabilities
```

Comment:
The URL on the rhs should probably be `reg/cadc/sandbox` instead of `reg/maq/sandbox` -- the `maq` one only gives datalink and metata capabilties and not soda (cutouts) because for maq we don't have the files.

Looks like test/GEMINI record should also be changed since we do have the files and in principle could be testing cutouts in the sandbox. I'd probably move them both out of the MAQ section as well for clarity.

Reply:
Done.